clients/include/testing_gemv_batched.hpp

```diff
@@ -46,23 +46,32 @@ hipblasStatus_t testing_gemv_batched(const Arguments& argus)
         dim_y = N;
     }
 
-    hipblasStatus_t status = HIPBLAS_STATUS_SUCCESS;
+    hipblasLocalHandle handle(argus);
 
     // argument sanity check, quick return if input parameters are invalid before allocating invalid
     // memory
-    if(M < 0 || N < 0 || lda < 0 || incx <= 0 || incy <= 0 || batch_count < 0)
-    {
-        return HIPBLAS_STATUS_INVALID_VALUE;
-    }
-    else if(batch_count == 0)
+    bool invalid_size = M < 0 || N < 0 || lda < M || lda < 1 || !incx || !incy || batch_count < 0;
```

Comment:
Curious to know if  `lda < 1` is required?

Reply:
This is directly from rocBLAS, so it appears that in the case that  `M == 0 && lda == 0`, we return `rocblas_status_invalid_size` as 0 is not an acceptable `lda`. However, if `M == 0 && lda > 0`, then we "quick return" `rocblas_status_success`.